lib/helper/FileSystem.js

```diff
@@ -136,7 +170,34 @@ class FileSystem extends Helper {
 function getFileContents(file, encoding) {
   if (!file) assert.fail('No files were opened, please use seeFile action');
   encoding = encoding || 'utf8';
-  return fs.readFileSync(file, 'utf8');
+  return fs.readFileSync(file, encoding);
```

Comment:
You changed this to encoding but the doc is still referring utf8?

Reply:
this is a private function not mentioned in the docs at all.

have a look at the line above and all the public methods. 'utf8' is the default encoding that may be changed now (it look like you may change it before but was hard coded)